src/components/Layout/Header/ActionCenter/components/SwapActionCard.tsx

```diff
@@ -59,28 +53,34 @@ export const SwapActionCard = ({
     return swapsById[action.swapMetadata.swapId]
   }, [action, swapsById])
 
-  const handleClick = useCallback(() => {
-    if (isCollapsable) {
-      onToggle()
-    }
-  }, [isCollapsable, onToggle])
+  const { isOpen, onToggle } = useDisclosure({ defaultIsOpen: swap.isStreaming })
 
   const hoverProps = useMemo(
     () => ({
-      bg: 'background.button.secondary.hover',
-      cursor: swap?.txLink ? 'pointer' : undefined,
+      bg: swap?.txLink ? 'background.button.secondary.hover' : 'transparent',
+      cursor: swap?.txLink ? 'pointer' : 'default',
+      textDecoration: 'none',
     }),
     [swap?.txLink],
   )
 
+  const handleClick = useCallback(() => {
+    if (swap.isStreaming) {
+      onToggle()
+    }
```

Comment:
q: this is tight-coupling toggle-able status to streaming swaps, which reading this is not very clear what this is doing, but I'm assuming this is because at the moment, only streaming swaps have a toggle state?
If so, this feels like a magic coupling we may not want, especially given we may want to add similar expanded state to other swaps with more info (e.g swap status string, or anything else we may feel like it)

Reply:
Makes sense, added back a isCollapsable prop and consumed it in [53df27f](https://github.com/shapeshift/web/pull/9695/commits/53df27f350df1a851f51947e6180239eb47782f0)